debs/SPECS/4.0.0/wazuh-agent/debian/postinst

```diff
@@ -133,6 +133,20 @@ case "$1" in
 
     # Restoring file permissions
     ${SCRIPTS_DIR}/restore-permissions.sh > /dev/null 2>&1 || :
+    
+    # Fix /etc/ossec-init.conf
+    chmod 640 /etc/ossec-init.conf
+    chown root:ossec /etc/ossec-init.conf
+
+    if [ ! -z "$2" ]; then
+        if [ -f ${WAZUH_TMP_DIR}/wazuh.restart ] ; then
+            if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1 ; then
+                systemctl restart wazuh-manager.service > /dev/null 2>&1
+            elif command -v service > /dev/null 2>&1 ; then
+                service wazuh-manager restart > /dev/null 2>&1
+            fi
+        fi
```

Comment:
To remove any possibility of having lingering files, remove the temp file in this block before restarting the service.

Reply:
The whole `WAZUH_TMP_DIR` directory is removed at the end of the script:

https://github.com/wazuh/wazuh-packages/blob/89c61b7696ecf7e98ff80ba52ee35cf901b45688/debs/SPECS/4.0.0/wazuh-agent/debian/postinst#L157-L159